package-lock.json

```diff
@@ -0,0 +1,16086 @@
+{
+  "name": "@nethermindeth/warp",
```

Comment:
can you delte this?
This is a npm artifact

Reply:
you mean the file?

Reply:
you mean the file?